src/Page/Guides/View.elm

```diff
@@ -37,16 +37,17 @@ view model =
 
                     Success list ->
                         List.concat [ Page.Guides.Data.teaserListFromGuideDict model.language model.content.guides, list ]
+
+        headerText : String
+        headerText =
+            if String.length model.query > 0 then
+                t <| GuidesTitleFiltered (String.fromInt <| List.length model.search) model.query
+
+            else
+                t GuidesTitle
     in
     div [ css [ centerContent ] ]
-        [ h1 [ attribute "aria-live" "alert" ]
```

Comment:
I think we're losing this attribute we need for screen readers from this page - maybe you could create another component, `searchResultPrimaryText` or something like that to be able to preserve this?